card/models.py

```diff
@@ -0,0 +1,33 @@
+from django.contrib.auth.models import User
+from django.db import models
+from django.db.models import UniqueConstraint
+from django.utils.translation import gettext_lazy as _
+
+
+class Card(models.Model):
+    """
+    Model for connecting a card number to a user
+    """
+    number = models.fields.CharField(max_length=16, verbose_name=_("Card number"))
+    user = models.OneToOneField(User, on_delete=models.CASCADE, verbose_name=_("User"))
+
+    class Meta:
+        constraints = (
+            UniqueConstraint(fields=['number'], name='unique_number'),
```

Comment:
This way of creating unique constraints is meant for more complicated functionality. For just making sure that a single field has unique values, just use `unique=True` when declaring the field. It also makes the code more readable.

Reply:
I think it would, however, be useful to create custom constraints for what values the number field can take. Such as being of a certain length and just being numbers.

Reply:
Regarding constraining the `number` field to only contain numbers: we could try using the [int-list-validator](https://docs.djangoproject.com/en/2.2/ref/validators/#int-list-validator) with `sep=''`..?

Reply:
I havn't looked at this for too long, but `int-list-validator` seems a bit complex. Also, just on a whim I think it might be good to have a more strict constraint. Such as numbers of X or Y digits. This, as we know that EM numbers have to be of a specific length (or two specific sizes when we also consider the old numbers).

Reply:
Old numbers can be zero-padded to get 10 digit numbers for them as well. The numeric value works the same way as the new ones and can be read by the card reader in the same way. Something like `\d{10}´ could work.

Reply:
Currently implemented the validation with a `RegexValidator` with the pattern `^\d{10}$`.